mteb/abstasks/AbsTaskInstructionRetrieval.py

```diff
@@ -427,7 +425,8 @@ def evaluate(self, model, split="test", **kwargs):
                         "short_instructions": scores_w_short_instr,
                         "base": scores_base[lang],
                     }
-        else:
+        else:  # seems like these two can be combined into one (with the new lang="default")
```

Comment:
@orionw, it seems like this if -else block and be simplified away, just checking before I do it to see if I missed something

Reply:
Sorry for the delay, was out of office for the weekend.  This else statement was to handle the different languages, so if we've combined that in other abstract tasks we should be able to combine it here also.

We don't have any non-English tasks implemented in it rn, but hopefully will have it soon.  You can remove now or I can test and remove when we add multilingual versions.

Reply:
I will leave it in for now then